tests/src/Unit/OgPermissionHandlerTest.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\og\Unit\OgPermissionHandlerTest.
+ */
```

Comment:
I would add a reference to the core class it was copy/ pasted and adjusted from.